src/main/java/emissary/pickup/file/FilePickUpClient.java

```diff
@@ -89,7 +89,7 @@ public FilePickUpClient(InputStream configInfo) throws IOException {
      */
     protected void configurePlace() {
         try {
-            digest = MessageDigest.getInstance("MD5");
+            digest = MessageDigest.getInstance("SHA-256");
```

Comment:
Should we make this a configurable value?

Reply:
I don't know where we would need to have it different. But it is easy enough to add so will do that.

Reply:
If it is not needed, don't worry about it. I didn't know if you'd need to change it internally without having to do a code change.